e2e/__tests__/components/funnelback-search-test.js

```diff
@@ -20,7 +20,7 @@ describe('SWE Header testing', () => {
     await page.type('input#qg-search-query', 'jobs', { delay: 40 });
     await page.waitForTimeout(ct.WT);
     expect(
-      await page.evaluate('$(\'.qg-search-concierge-content li button\')')).toHaveLength(3);
+      await page.evaluate('$(\'.qg-search-concierge-content li button\').length')).toHaveLength(3);
```

Comment:
Wouldn't this mean that 'length' is a 3-digit number?

Reply:
I just changed toHaveLength with toBe and the test passed.